docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -2506,7 +2507,7 @@ from **200 OK** to **101 UPGRADED** and resends the same headers.
 
 ## 3.3 CORS Requests
 
-To set cross origin requests to the remote api please give values to 
+To set cross origin requests to the remote api please give values to
```

Comment:
I built the docs to investigate this. I think most of these are unnecessary (the list handles things just fine).  This one however I can see being useful to prevent the line wrap of the command below.

I'll revert these changes and disable this pre-commit for markdown files.